x/wasm/types/errors.go

```diff
@@ -69,4 +69,23 @@ var (
 
 	// ErrInvalidEvent error if an attribute/event from the contract is invalid
 	ErrInvalidEvent = sdkErrors.Register(DefaultCodespace, 21, "invalid event")
+
+	// ErrNoSuchContract error if an address does not belong to a contract
```

Comment:
Why do we need this if we have the custom type? Just to ensure no one else uses it?

If so, I would name it `errNoSuchContract` or such to make it private, just register it